pmsoService/product_flow/features/import_request/api/views/list_create_views.py

```
@@ -59,6 +61,15 @@ def create(self, request, *args, **kwargs):
         """
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
+
+        # Check status of sale order
+        sale_order = serializer.validated_data.get('sale_order') if hasattr(serializer, 'validated_data') else None
+        if sale_order:
+            try:
+                validate_sale_order_allows_import_request_changes(sale_order.status)
+            except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)